packages/orbit-components/src/Tag/index.tsx

```diff
@@ -233,48 +37,76 @@ const Tag = React.forwardRef<HTMLDivElement, Props>(
     ref,
   ) => {
     return (
-      <StyledTag
-        className="orbit-tag"
-        actionable={!!(onClick || onRemove)}
+      <div
+        className={cx(
+          "orbit-tag",
+          "font-base rounded-large p-xs box-border inline-flex items-center justify-center font-medium",
+          "duration-fast transition-[color,_background-color,_box-shadow] ease-in-out",
+          "tb:rounded-normal",
+          size === SIZES.SMALL && "text-small leading-small",
+          size === SIZES.NORMAL && "text-normal leading-normal",
+          !!(onClick || onRemove) &&
+            "cursor-pointer [&_.orbit-tag-close-container]:active:opacity-100",
+          selected && [
+            "text-white-normal",
+            dateTag
+              ? "bg-ink-light-hover hover:bg-ink-light-active focus:bg-ink-light-active active:bg-ink-light-hover"
+              : "bg-blue-normal hover:bg-blue-normal-hover focus:bg-blue-normal-hover active:bg-blue-normal-active",
+          ],
+          type === TYPES.NEUTRAL && [
+            "text-ink-dark",
+            !selected &&
+              "bg-cloud-normal hover:bg-cloud-normal-hover focus:bg-cloud-normal-hover active:bg-cloud-normal-active",
+          ],
+          type === TYPES.COLORED && [
+            "text-blue-darker",
+            !selected &&
+              "bg-blue-light hover:bg-blue-light-hover focus:bg-blue-light-hover active:bg-blue-light-active",
+          ],
+        )}
         data-test={dataTest}
         id={id}
-        dateTag={dateTag}
-        size={size}
         ref={ref}
-        type={type}
         onClick={onClick}
-        removable={!!onRemove}
-        selected={selected}
         tabIndex={(onClick || onRemove) && 0}
         role="button"
-        onKeyDown={buttonClickEmulation(onClick)}
+        onKeyDown={ev => buttonClickEmulation(ev, onClick)}
       >
-        {iconLeft && <StyledIconContainer>{iconLeft}</StyledIconContainer>}
+        {iconLeft && (
+          <div className="pr-xs [&_svg]:size-icon-small flex flex-row items-center justify-center">
+            {iconLeft}
+          </div>
+        )}
         {children}
         {onRemove && (
-          <CloseContainer
-            selected={selected}
-            removable={!!onRemove}
-            type={type}
+          <div
+            className={cx(
+              "orbit-tag-close-container",
+              "ms-xs rounded-circle flex",
+              "duration-fast transition-[color,_opacity] ease-in-out",
+              "focus:opacity-100",
+              !selected &&
+                (type === TYPES.NEUTRAL
+                  ? "text-ink-normal active:text-ink-dark"
+                  : "text-blue-darker"),
+              selected ? "text-white-normal opacity-100" : "opacity-50",
+            )}
+            tabIndex={0}
+            aria-label="close"
+            role="button"
```

Comment:
cursor pointer is present even when onClick is not applied, I believe this happens because of `role="Button"`, can we not have a role="button" when onClick is not applied? just a regular div, without tabIndex and role